Sources/Orbit/Support/Layout/IgnoreScreenLayoutHorizontalPaddingModifier.swift

```diff
@@ -0,0 +1,66 @@
+import SwiftUI
+
+struct IgnoreScreenLayoutHorizontalPaddingModifier: ViewModifier {
+
+    @Environment(\.horizontalSizeClass) var horizontalSizeClass
+    @Environment(\.screenLayoutPadding) var screenLayoutPadding
+
+    func body(content: Content) -> some View {
+        content
+            .padding(.horizontal, horizontalPadding)
+    }
+
+    var horizontalPadding: CGFloat {
+        guard horizontalSizeClass != .regular, let screenLayoutPadding = screenLayoutPadding else {
+            return 0
+        }
+
+        return -screenLayoutPadding.horizontal(horizontalSizeClass: horizontalSizeClass)
+    }
+}
+
+public extension View {
+
+    /// Reverts any horizontal padding provided by `screenLayout` context.
+    ///
+    /// A typical usage is to mimic the edge-to-edge appearance of a `Card` component.
```

Comment:
Shall we mention in the description, that it's valid only for `compact` mode?

Reply:
Do you mean `compact` width (interface size) or `compact` padding (one of padding cases)?

Reply:
I meant compact width.
We do check `horizontalSizeClass != .regular` and if it's `horizontalSizeClass == .regular` we appply zero padding.

I suggest reflecting that in  docs

Reply:
I am probably missing something. This modifier is independent on it, it specifies that it "reverts **any** horizontal padding", meaning "if it was applied, revert it". 
The call site cannot optionally use this modifier, it needs to use it all the time anyway.

Reply:
Thx, I finally get it! 👍  
I wanted to even rename it to reflect that, but found no good name. Updated the doc comments.